buildSrc/src/main/groovy/com/github/okbuilds/okbuck/generator/BuckFileGenerator.groovy

```diff
@@ -133,6 +133,12 @@ final class BuckFileGenerator {
             ":${rule.name}"
         }
 
+        // SqlDelight
+        if (target.sqldelight != null) {
+            rules.add(GenSqlDelightRuleComposer.compose(target))
+            deps.add(":sqldelight_${target.name}")
```

Comment:
i dont think this will work.

the sqldelight genrule needs to run before the src_target.name rule is built.

You may need to extract all sql files to a separate android module to make this work correctly